types/events.go

```diff
@@ -210,15 +223,9 @@ func (e Events) ToABCIEvents() []abci.Event {
 	return res
 }
 
-func toBytes(i interface{}) []byte {
-	switch x := i.(type) {
-	case []uint8:
```

Comment:
I assume this case is never actually used?

Reply:
Yeah, and it was bad to be casting to interface and then reflecting.